### PR TITLE
chore(windows): remove obsolete importkeyboard app

### LINKED ISSUE #4042: chore(windows): buildtools/importkeyboard should be removed
importkeyboard has been replaced by kmconvert.

### PR DESCRIPTION
**Pull request by mcdurdin**
Functionality has been ported to kmconvert.

Fixes #4042.

### CODE REVIEW

File: windows/src/TODO.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# TODO Monday 2020-12-14
+
+1. Keyboard icons still not showing in Keyman Configuration keyboards tab
+2. Some keyboards not activating through keyman menu/toolbox toolbar -- amharic, greek, hebrew
+3. Same keyboards also not showing on osk
+4. After removing a package, Keyman OSK toolbar is not updated to reflect removed packages
+5. After installing a package, Keyman OSK toolbar is not refreshed until we disable/enable a keyboard
+6. Suggestion: the 'repair profiles' pattern should look at the list of enabled Keyman keyboards vs the list of profiles in HKCU\Control Panel\International and fixup discrepancies there. This should probably be an automatic task. This will probably fix the issues that Makara noted around missing keyboards across multiple user profiles etc.
```
Comment (darcywong00) [inline windows/src/TODO.md:8]: Are the new TODO's intentional?

Comment (mcdurdin) [reply windows/src/TODO.md:8]: Ah, no , I will pull those out again!

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.9-alpha](https://keyman.com/downloads/releases/alpha/15.0.9)

**Pull request merged**